### DISCUSSION TITLE
md file looses style if it has been pushed by the CLI

### DISCUSSION THREAD

**Discussion by sandorferencz** [structurizr/onpremises#48] [Q&A] [answered]
![image](https://github.com/structurizr/dsl/assets/6851338/2b9c016b-44a0-4165-a90c-b5d0056c5073)
 

As per screenshot a meaningful style section has been added to the md documentation. While the CLI has been used to push the workspace.dsl to the on-premise url the documentation turned the style into a text papering on the page see second screenshot. 

![image](https://github.com/structurizr/dsl/assets/6851338/9f6f315c-d07a-47b7-9a06-4fd39acbff4a)

Obviously these are only images so I can resize them, I am just not sure if I miss something or it is really an issue what needs attention some point. 

Thanks for looking at it in advance. 
Best Regards
Sandor

**Comment by simonbrowndotje** [comment, marked as answer]
You could try setting `structurizr.safeMode` to `false` (see https://structurizr.com/share/18571/documentation#structurizrproperties), but I don't know whether the Markdown renderer being used supports HTML/CSS.

**Reply by sandorferencz** [reply]
Hi Simon,

Thanks for the reply, yes once we turn the `structurizr.safeMode` to false it allows the <style></style> to remain as hidden in the markdown and works correctly. 
Maybe it would be nice to allow this to be pushed without turning the safemode off. I feel this is a nice to have not a must feature. 

Otherwise great work I live your product many thanks for bringing it live.

**Reply by DuncanvR** [reply]
@simonbrowndotje On the new (?) documentation page (https://docs.structurizr.com/onpremises/configuration), the `safeMode` setting is no longer mentioned. Was it removed?

It's still listed behind the link you mentioned, which makes it confusing which page is correct. Other settings, like `structurizr.feature.diagramReviews`, are missing, so I'm guessing the old page simply isn't updated any more. Should it be taken offline, perhaps? Or kept, but only contain a link to the new documentation?
